config/config-example.js

```diff
@@ -171,6 +171,11 @@ exports.crashguardemail = null;
  */
 exports.disablebasicnamefilter = false;
 
+/**
+ * allowrequestingties - disables the use of `/offerdraw` and `/acceptdraw`
```

Comment:
Is this comment right? Its description sounds like it contradicts the variable name to me?

Reply:
Yeah, I noticed it when testing.